truffle.js

```diff
@@ -1,12 +1,12 @@
 require("babel-register");
 require("babel-polyfill");
+const TestRPC = require("ethereumjs-testrpc");
 
 module.exports = {
   networks: {
     development: {
-      host: "localhost",
-      port: 8545,
-      network_id: "*" // Match any network id
+      network_id: "*",
+      provider: TestRPC.provider()
```

Comment:
This helps a lot. Eliminates all (slow) http traffic during test. Unfortunately does not get used when runing solcover.